management/ia/channels_test.go

```diff
@@ -13,6 +13,9 @@ import (
 	pmmapitests "github.com/Percona-Lab/pmm-api-tests"
 )
 
+// Note: Even though the IA services check for alerting enabled or disabled before returning results
```

Comment:
added this comment because IA tests are a bit different from others like STT checks where we enable and disable checks through settings for every test case, let me know if its ok to keep this